doc/shield-coinbase.md

```diff
@@ -0,0 +1,101 @@
+# Shielding Coinbase UTXOs
+
+**Summary**
+
+Use z_shieldcoinbase RPC call to shield coinbase utxos.
```

Comment:
Here "utxos" is lowercase, but it's "UTXOs" elsewhere. Keep capitalization consistent, probably changing to "UTXOs" everywhere is best.

Reply:
ok